en/README.md

```diff
@@ -114,7 +114,7 @@ Select the links below to display the calendar in your timezone (and to add it t
 * [Australia – Melbourne/Sydney/Hobart](https://calendar.google.com/calendar/embed?src=linuxfoundation.org_g21tvam24m7pm7jhev01bvlqh8%40group.calendar.google.com&ctz=Australia%2FSydney)
 
 :::tip 
-Calendar defaults to CET.
+Calendar shown defaults to CET, as CET is the standard timezone used for defining the meeting times (Since different timezones have different daylight saving policies, we chose CET as singular source of truth)

```

Comment:
I don't think you need to explain the why here. Who cares?

```suggestion
The calendar default timezone is Central European Time (CET).

```

Reply:
True! It's just a detail 🙏